src/vtquery.cpp

```diff
@@ -219,6 +231,108 @@ std::vector<vtzero::property> get_properties_vector(vtzero::feature& f) {
     return v;
 }
 
+double convert_to_double(value_type value) {
```

Comment:
Ran into type conversion errors when the filter was one type (float) and the parameter value was a different type (int64)